README.md

```diff
@@ -1,273 +1,119 @@
-# strong-pm
+# strong-pm - Process Manager
 
-StrongLoop process manager.
+[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/strongloop/chat?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-## Installation
-
-    npm install -g strong-pm
-
-## Quick Start
+StrongLoop's process manager, strong-pm, allows you to build, deploy, monitor,
+and debug your apps in development and in production.
 
-It is recommend to install the process manager as a system service, see below,
-but if you are just trying the manager out to try it, it can be run directly
-from the command line.
+For more details, see http://strong-pm.io.
 
-Run process manager:
+## Features
 
-    sl-pm
+- Build, package, and deploy your Node application to a local or remote system.
+- Manage processes and cluster (use all the cores on your server), adjust sizes, built-in load balancer
+- Automatic restarts keep applications running forever
+- Zero-downtime redeploy/upgrade
+- View performance metrics on your application
+- View CPU profiles and heap snapshots to optimize performance and diagnose memory leaks
+- Use graphical tool StrongLoop Arc, **slc arc**, or command-line tool, **slc**
 
-Install [strong-build](http://github.com/strongloop/strong-build) and
-[strong-deploy](http://github.com/strongloop/strong-deploy):
+***XXX We should put the comparison table right here methinks***
 
-    npm install -g strong-build
-    npm install -g strong-deploy
+### Build & Deploy
 
-Clone, build, and deploy an app, the loopback example app in this example, but
-any node application can be managed:
+- Automated build and multi-host deploy.
+- Zero-downtime application restarts and upgrades.
+- Install dependencies, run custom build steps, and prune development dependencies without affecting your source tree.
+- Deploy without dependence on outside servers (like NPM)
+- Create an npm package or commit the build onto a Git deploy branch.
 
-    git clone git@github.com:strongloop/loopback-example-app.git
-    cd loopback-example-app
-    sl-build
-    sl-deploy
+### Profile
 
-## Life-cycle
+- Heap snapshots and CPU profiles.
+- Profile local or remote applications with Arc.
+- Profile local applications with slc.
+- Use the _watchdog_ to start CPU profiling when the Node event loop stalls.
+  ***XXX we should really remove this feature and only make it available through
+  Arc... different issue though... maybe just remove for now.***
```

Comment:
@ijroth watchdog requires a specific watchdog license, should be good enough, shouldn't it?